### PR TITLE
#20 Added email paging functionality with button that triggers this

### LINKED ISSUE #20: Email Paging
Have the ability to load more than the default number of emails on the inbox page.

Notes:

Acceptance Criteria:

- [x] Paging of functionality.
- [x] UI works.

### PR DESCRIPTION
**Pull request by D3ISM3**
ezgif-6-c0ac0ece5fbe

Added paging.

Technically speaking, this just gets the count of emails that we currently have and then increments that number by 10 and display the new emails that have been pulled.

The functionality is there, but an improvement for the future to consider is actually making this process more efficient.

### CODE REVIEW

File: ec_email_client/src/pages/InboxPage.js [hunk 1]
```diff
@@ -122,6 +122,12 @@ function InboxPage(props) {
         return atob(data);
     }
 
+    function Test() {
```
Comment (jameswatters1) [inline ec_email_client/src/pages/InboxPage.js:125]: May get a little confusing naming this Test but I'm not a picky person

**Review by jameswatters1 — APPROVED** [review]
I tested it and it works for me

File: ec_email_client/src/pages/InboxPage.js (hunk 1 above)
Comment (pbell97) [reply ec_email_client/src/pages/InboxPage.js:125]: Yeah this is the only thing I'd have any concerns with. We can change it in future if we need to hurry this PR in though...

**Review by pbell97 — APPROVED** [review]
Just tested, works great.

**Review by nenntendo64 — APPROVED** [review]

**Pull request merged**